### PR TITLE
Bug 1850140: tests: extended: fix TM regression test

### PR DESCRIPTION
**Pull request by fromanirh**
TL;DR: the topology manager code is correct, a specific test fails to capture this leading to false negatives

The existing regression test which wants to cover against regressions
on RHBZ#1813567 is actually doing the reverse check than it should.
We noticed after having the test run for few days on environment kindly
provided by nfvpe team.
The likely cause of why this mistake sneaked in is that I wrongly
configured the original environment on which I tried the test.

This patch fixes the issue and puts extra care in documenting and
checking the intent of the original bugzilla.

Signed-off-by: Francesco Romani <fromani@redhat.com>

### CODE REVIEW

**Comment by fromanirh** [comment]
verified on 4.4 (fails as expected), pending verification on 4.5

File: test/extended/topology_manager/resourcealign.go [hunk 1]
```diff
@@ -260,6 +260,11 @@ var _ = g.Describe("[Serial][sig-node][Feature:TopologyManager] Configured clust
 			})
 
 			// rhbz#1813567
+			// a well-behaving TM must *NOT* admit either of the pods.
```
Comment (vpickard) [inline test/extended/topology_manager/resourcealign.go:263]: Should this be, a well-behaving TM must admin only 1 of the pods?

Comment (fromanirh) [reply test/extended/topology_manager/resourcealign.go:263]: Yep, bad wording. Will rephrase and fix.

**Comment by fromanirh** [comment]
had a few runs against a 4.5 cluster, works as expected

**Comment by fromanirh** [comment]
3rd parthy verification: https://bugzilla.redhat.com/show_bug.cgi?id=1826807

**Comment by fromanirh** [comment]
ping?

**Comment by rphillips** [comment]
/retitle Bug 1850140: tests: extended: fix TM regression test
